examples/client/Locomotion/src/lib/ride/utils.ts

```diff
@@ -16,15 +20,28 @@ export const formatSps = function (stopPoints: any) {
     .map((sp: any) => ({
       ...sp,
       ordinalDesc: getOrdinalDescForSp(sp),
+    }))
+    .map((sp: any) => ({
+      ...sp,
+      amountOfSpsFromType: orderDesc[sp.type],
     }));
 };
 
+
 export const getOrdinal = (n: number) => {
   const s = ['th', 'st', 'nd', 'rd'];
   const v = n % 100;
   return n + (s[(v - 20) % 10] || s[v] || s[0]);
 };
 
+export const formatSpText = (sp: any) => {
+  const uiSpType = formatUiDisplaySpType(sp, null);
+  const { ordinalDesc } = sp;
+  const isMultiSp = isMulti(sp);
+  const isOnlyMultiSp = isMultiSp && sp.amountOfSpsFromType === 2;
+  const numberPart = `${sp.ordinalDesc && !isOnlyMultiSp ? `${getOrdinal(isMultiSp ? ordinalDesc : ordinalDesc + 1)} ` : ''}`;
```

Comment:
what is numberPart ? can you change it to more readable name (or add comment)? 🙏  essentially I don't understand the output of formatSpText method